discord/ext/ipc/errors.py

```diff
@@ -1,7 +1,36 @@
+"""
+    Copyright 2020 Ext-Creators
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+
 class NoEndpointFoundError(Exception):
```

Comment:
The exceptions you define should all derive from a single extension-wide exception. `IPCError` perhaps.